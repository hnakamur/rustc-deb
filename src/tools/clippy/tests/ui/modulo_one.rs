#![warn(clippy::modulo_one)]
#![allow(clippy::no_effect, clippy::unnecessary_operation)]

fn main() {
    10 % 1;
    10 % 2;
}
