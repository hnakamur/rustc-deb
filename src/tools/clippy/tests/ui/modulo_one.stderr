error: any number modulo 1 will be 0
  --> $DIR/modulo_one.rs:5:5
   |
LL |     10 % 1;
   |     ^^^^^^
   |
   = note: `-D clippy::modulo-one` implied by `-D warnings`

error: aborting due to previous error

